src/frontend/src/lib/components/swap/SwapDetailsIcp.svelte

```diff
@@ -4,28 +4,36 @@
 	import ModalValue from '$lib/components/ui/ModalValue.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { SWAP_CONTEXT_KEY, type SwapContext } from '$lib/stores/swap.store';
+	import type { OptionAmount } from '$lib/types/send';
 	import type { SwapMappedResult, SwapProvider } from '$lib/types/swap';
-	import { formatToken } from '$lib/utils/format.utils';
+	import { formatReceiveOutMinimum } from '$lib/utils/swap.utils';
 
-	const { provider } = $props<{
+	interface Props {
 		provider: Extract<SwapMappedResult, { provider: SwapProvider.ICP_SWAP }>;
-	}>();
+		slippageValue: OptionAmount;
```

Comment:
since it can be undefined, even optional? or better not? it doesn't make sense, right?

in case yes, change it for the other components too pls

```suggestion
		slippageValue?: OptionAmount;
```

Reply:
it doesn't make any sense as OptionAmount is
```
export type Amount = string | number;
export type OptionAmount = Amount | undefined;
```